code/modules/admin/admin.dm

```diff
@@ -29,12 +29,37 @@ proc/admin_notice(var/message, var/rights)
 /datum/admins/SDQL_update(var/const/var_name, var/new_value)
 	return 0
 
+/datum/admins/Topic(href, href_list)
+	..()
```

Comment:
Move this to `topic.dm` so it's easy to clean it up later.

Reply:
Now done!